tests/zfs-tests/tests/functional/cli_root/zpool_get/zpool_get_002_pos.ksh

```diff
@@ -48,19 +48,25 @@
 # default values are sane, or whether they can be changed with zpool set.
 #
 
+function cleanup
+{
+	rm $values
```

Comment:
IIRC `rm` without `-f` argument may ask for confirmation (for sure in rhel-like distros).

Reply:
thank you @gmelikov, i can fix this.